### PR TITLE
Add contests table that will be an aggregate under an election

### PR DESCRIPTION
**Pull request by dbolson**
@DSASanFrancisco/portal-members 

Closes https://github.com/DSASanFrancisco/membership_api/issues/24

### CODE REVIEW

File: alembic/versions/74c4e2071af2_add_contests_table.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+
+def upgrade():
+    op.create_table(
+        'contests',
+        sa.Column('id', sa.Integer(), nullable=False),
+        sa.Column('election_id', sa.Integer(), nullable=False),
+        sa.Column('status', sa.String(length=45), nullable=False),
+        sa.Column('number_winners', sa.Integer(), nullable=True),
+        sa.Column('count_rules', sa.String(length=254), nullable=False),
```
Comment (jeffmay) [inline alembic/versions/74c4e2071af2_add_contests_table.py]: I suppose we should put the `enum` rules in the code then, right?

Comment (dbolson) [inline alembic/versions/74c4e2071af2_add_contests_table.py]: That was my thinking. I copied the current `status` implementation in `elections` (not a good excuse), and I'm not particularly excited about how mysql handles enums,
 especially [invalid ones](https://dev.mysql.com/doc/refman/5.7/en/enum.html):

> If you insert an invalid value into an ENUM (that is, a string not present in the list of permitted values), the empty string is inserted instead as a special error value.

Comment (jessebutterfield) [inline alembic/versions/74c4e2071af2_add_contests_table.py]: I usually do this 
status = Column(Enum(MyEnum, native_enum=False, create_constraint=False))
which stores the enum as a string but allows sql_alchemy to convert it to an enum automatically. I also hate how mysql handles enums.

Comment (dbolson) [inline alembic/versions/74c4e2071af2_add_contests_table.py]: I'll play around with it and open a new PR.

**Pull request merged**